charts/k8s-monitoring/default_allow_lists/alloy_integration.yaml

```diff
@@ -1,57 +1,65 @@
 ---
 # The set of metrics from Grafana Alloy required for the Grafana Alloy integration
-- alloy_tcp_connections
-- alloy_wal_samples_appended_total
-- alloy_wal_storage_active_series
+- alloy_build_info
```

Comment:
`alloy_build_info` is already in by default so not needed here, although it doesn't do any harm either way.

```suggestion
```

Reply:
I'm fine with duplicating. This protects against things like:
```
metricsTuning:
  useDefaultAllowList: false
  useIntegrationAllowList: true
```